domain/generated_types.go

```diff
@@ -68,11 +68,21 @@ type Customer struct {
 // CustomersResponse defines model for CustomersResponse.
 type CustomersResponse []Customer
 
-// An endpoint registered on a DID Document.
+// Endpoint defines model for Endpoint.
 type Endpoint struct {
+	// Embedded struct due to allOf(#/components/schemas/EndpointID)
+	EndpointID `yaml:",inline"`
```

Comment:
I find it very ugly that the ID attribute of Endpoint is named `Endpoint.EndpointID` instead of just `Endpoint.ID`. Same goes for `EndpointProperties` and `Service`

Reply:
Its a result from serperating the ID from the Properties which allows us to combine the two when used in return values, but only the properties when used as post values.